{"uuid": "c3eca938-34ee-4b9e-bb81-c2fb691a0d08", "children": ["34eb6eaa-6f71-44a1-b4c9-6ac8c036e5f7"], "befores": [{"name": "setup", "status": "broken", "statusDetails": {"message": "selenium.common.exceptions.WebDriverException: Message: unknown error: net::ERR_INTERNET_DISCONNECTED\n  (Session info: chrome=123.0.6312.106)\nStacktrace:\n\tGetHandleVerifier [0x00007FF7A4C07032+63090]\n\t(No symbol) [0x00007FF7A4B72C82]\n\t(No symbol) [0x00007FF7A4A0EC65]\n\t(No symbol) [0x00007FF7A4A0AC1A]\n\t(No symbol) [0x00007FF7A49FD514]\n\t(No symbol) [0x00007FF7A49FE8F8]\n\t(No symbol) [0x00007FF7A49FD7FF]\n\t(No symbol) [0x00007FF7A49FD104]\n\t(No symbol) [0x00007FF7A49FD021]\n\t(No symbol) [0x00007FF7A49FB0A8]\n\t(No symbol) [0x00007FF7A49FB8BC]\n\t(No symbol) [0x00007FF7A4A114AD]\n\t(No symbol) [0x00007FF7A4A93D67]\n\t(No symbol) [0x00007FF7A4A76FDA]\n\t(No symbol) [0x00007FF7A4A93412]\n\t(No symbol) [0x00007FF7A4A76D83]\n\t(No symbol) [0x00007FF7A4A483A8]\n\t(No symbol) [0x00007FF7A4A49441]\n\tGetHandleVerifier [0x00007FF7A50025AD+4238317]\n\tGetHandleVerifier [0x00007FF7A503F70D+4488525]\n\tGetHandleVerifier [0x00007FF7A50379EF+4456495]\n\tGetHandleVerifier [0x00007FF7A4CE0576+953270]\n\t(No symbol) [0x00007FF7A4B7E54F]\n\t(No symbol) [0x00007FF7A4B79224]\n\t(No symbol) [0x00007FF7A4B7935B]\n\t(No symbol) [0x00007FF7A4B69B94]\n\tBaseThreadInitThunk [0x00007FFF7A33257D+29]\n\tRtlUserThreadStart [0x00007FFF7B7AAA58+40]\n\n", "trace": "  File \"C:\\Users\\nikhi\\PycharmProjects\\CredeKart_Pytest_Project\\.venv\\Lib\\site-packages\\pluggy\\_callers.py\", line 166, in _multicall\n    teardown.throw(outcome._exception)\n  File \"C:\\Users\\nikhi\\PycharmProjects\\CredeKart_Pytest_Project\\.venv\\Lib\\site-packages\\_pytest\\setuponly.py\", line 36, in pytest_fixture_setup\n    return (yield)\n            ^^^^^\n  File \"C:\\Users\\nikhi\\PycharmProjects\\CredeKart_Pytest_Project\\.venv\\Lib\\site-packages\\pluggy\\_callers.py\", line 102, in _multicall\n    res = hook_impl.function(*args)\n          ^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"C:\\Users\\nikhi\\PycharmProjects\\CredeKart_Pytest_Project\\.venv\\Lib\\site-packages\\_pytest\\fixtures.py\", line 1132, in pytest_fixture_setup\n    result = call_fixture_func(fixturefunc, request, kwargs)\n             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"C:\\Users\\nikhi\\PycharmProjects\\CredeKart_Pytest_Project\\.venv\\Lib\\site-packages\\_pytest\\fixtures.py\", line 913, in call_fixture_func\n    fixture_result = fixturefunc(**kwargs)\n                     ^^^^^^^^^^^^^^^^^^^^^\n  File \"C:\\Users\\nikhi\\PycharmProjects\\CredeKart_Pytest_Project\\testCases\\conftest.py\", line 41, in setup\n    driver.get(\"https://automation.credence.in\")\n  File \"C:\\Users\\nikhi\\PycharmProjects\\CredeKart_Pytest_Project\\.venv\\Lib\\site-packages\\selenium\\webdriver\\remote\\webdriver.py\", line 356, in get\n    self.execute(Command.GET, {\"url\": url})\n  File \"C:\\Users\\nikhi\\PycharmProjects\\CredeKart_Pytest_Project\\.venv\\Lib\\site-packages\\selenium\\webdriver\\remote\\webdriver.py\", line 347, in execute\n    self.error_handler.check_response(response)\n  File \"C:\\Users\\nikhi\\PycharmProjects\\CredeKart_Pytest_Project\\.venv\\Lib\\site-packages\\selenium\\webdriver\\remote\\errorhandler.py\", line 229, in check_response\n    raise exception_class(message, screen, stacktrace)\n"}, "start": 1712819543346, "stop": 1712819545876}], "start": 1712819543346, "stop": 1712819545922}